ts/api/backend.ts

```diff
@@ -226,8 +238,15 @@ export function BackendClient(
     response_decoder: getActivationStatusDefaultDecoder()
   };
 
+  const withBearerToken = <P extends { Bearer: string }, R>(
+    f: (p: P) => Promise<R>
+  ) => async (po: Omit<P, "Bearer">): Promise<R> => {
+    const params = Object.assign({ Bearer: String(token) }, po) as P;
+    return f(params);
+  };
+
   return {
-    getSession: createFetchRequestForApi(getSessionT, options),
+    getSession: withBearerToken(createFetchRequestForApi(getSessionT, options)),
```

Comment:
`getSession` now doesn't require the `Bearer` parameter anymore - but it still gets passed to the function created by `createFetchRequestForApi`